tests/testthat/_snaps/build-home-index.md

```diff
@@ -70,23 +76,23 @@
 
 # data_home_sidebar() outputs informative error messages
 
-    Can't find component home.sidebar.components.fancy in '_pkgdown.yml'.
+    Can't find home.sidebar.components.fancy component in '_pkgdown.yml'.
```

Comment:
IMO component first reads better

Reply:
If I move component up then I lose the [ability to pluralize](https://cli.r-lib.org/articles/pluralization.html?q=qty#choosing-the-right-quantity) "component" because there are ambiguous glue subs, and I'd like to keep the file link:

> Note that if the message only contains a single [{}](https://rdrr.io/r/base/Paren.html) substitution, then this may appear before or after the pluralization markup. If the message contains multiple [{}](https://rdrr.io/r/base/Paren.html) substitutions after pluralization markup, an error is thrown.

Reply:
Hmmm, good point. Looking at the error more, maybe we can move `home.sidebar.components` later? i.e. more like "Can't find 'fancy' component in `home.sidebar.components` in `_pkgdown.yml`"?